pkg/utils/mountinfo/mountpoint.go

```diff
@@ -64,14 +69,15 @@ func getGlobalMounts(mountByPath map[string]*Mount) (globalMountByName map[strin
 
 	for k, v := range mountByPath {
 		if strings.Contains(k, mountRoot) {
+			vCopy := v
```

Comment:
I think vCopy := v does not create a deep copy of the Mount object; instead, it creates a new pointer to the same Mount object. If the original Mount object is modified, vCopy will reflect these changes. If this behavior is not intended, implementing a method for deep copying might be necessary.